tests/e2e/idps_test.go

```diff
@@ -5,13 +5,19 @@ import (
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	CI "github.com/terraform-redhat/terraform-provider-rhcs/tests/ci"
 	conn "github.com/terraform-redhat/terraform-provider-rhcs/tests/ci"
 	"github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/cms"
 	con "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/constants"
 	exe "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/exec"
+	"github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/openshift"
 )
 
 var _ = Describe("TF Test", func() {
+	var clusterID string
+	BeforeEach(func() {
```

Comment:
Why do we need to get cluster ID before every test case? why not to get in in the "Before suit" - in init_test.go?

Reply:
Because not all cases need a cluster ID. Especially some day1 negative cases, there is no cluster created at all.

Reply:
But most of the cases will be Day2.  
If I understand the flow - the cluster will be created anyway by profile in the flow, but not every test case will use it. Looks for me overwhelming to set it for every Before Each, instead of - once per test suit

Reply:
@xueli181114 I believe there won't be any error if there won't be any use for it for some cases,
so it can still be in the `init_test.go` IMO

Reply:
OK. I checked terraform output won't return error when there is no resources creation. Moving it back to init_test.go

Reply:
Updated. It had been moved back to init_test.go